extensions/interactions/NumberWithUnits/directives/number_with_units_help_modal_directive.html

```diff
@@ -0,0 +1,149 @@
+        forward slash '/' symbol.
+      </td>
+      <td>3 by 5 km</td>
+      <td>3/5 km</td>
+    </tr>
+    <tr>
+      <td>
+        For multiplying two or more units, put one space or asterisk '*' between them.
+      </td>
+      <td>5 J-s <br>  5 J . s</td>
+      <td>5 J s <br> 5 J * s</td>
+    </tr>
+    <tr>
+      <td>
+        For adding dimensions to the unit, use caret '^' symbol to add both positive
+        and negative powers.
+      </td>
+      <td>5 kg / m(2) <br> 5 kg / m**2 <br> 5 kg m^(-2)</td>
+      <td>5 kg / m^2 <br> 5 kg / m^2 <br> 5 kg m^-2</td>
+    </tr>
+    <tr>
+      <td>
+        In case of multiple units in the denominator, enclose them within the parenthesis.
+      </td>
+      <td>5 kg / m * s <br> 5 kg/m*s</td>
+      <td>5 kg / (m * s) <br> 5 kg / m / s</td>
+    </tr>
+    <tr>
+      <td>
+        For writing beginning units (like currencies), separate unit and value with one space.
+      </td>
+      <td>$5 <br> 5$ <br> 5 $</td>
+      <td>$ 5</td>
+    </tr>
+  </table><br><br>
+
+  <table class="table-bordered">
+    <tr>
+      <td><b>Base</b></td>
+      <td><b>Unit</b></td>
+    </tr>
+    <tr>
+      <td>Length</td>
+      <td>meter (m), inch (in), foot (ft), yard (yd), mile (mi), link (li), rod (rd), chain (ch), angstrom, mil</td>
+    </tr>
+    <tr>
+      <td>Surface area</td>
+      <td>m2, sqin, sqft, sqyd, sqmi, sqrd, sqch, sqmil, acre, hectare</td>
+    </tr>
+    <tr>
+      <td>Volume</td>
+      <td>m3, litre (l, L, lt, liter), cc, cuin, cuft, cuyd, teaspoon, tablespoon</td>
+    </tr>
+    <tr>
+      <td>Angles</td>
+      <td>rad (radian), deg (degree), grad (gradian), cycle, arcsec (arcsecond), arcmin (arcminute)</td>
+    </tr>
+    <tr>
+      <td>Time</td>
+      <td>
+        second (s, secs, seconds), minute (mins, minutes), hour (h, hr, hrs, hours), day (days),
+        week (weeks), month (months), year (years), decade (decades), century (centuries), millennium (millennia)
+      </td>
+    </tr>
+    <tr>
+      <td>Frequency</td>
+      <td>hertz (Hz)</td>
+    </tr>
+    <tr>
+      <td>Mass</td>
+      <td>
+        gram(g), tonne, ton, grain (gr), dram (dr), ounce (oz), poundmass (lbm, lb, lbs),
+        hundredweight (cwt), stick, stone
+      </td>
+    </tr>
+    <tr>
+      <td>Electric current</td>
+      <td>ampere (A)</td>
+    </tr>
+    <tr>
+      <td>Temperature</td>
+      <td>kelvin (K), celsius (degC), fahrenheit (degF), rankine (degR)</td>
+    </tr>
+    <tr>
+      <td>Amount of substance</td>
+      <td>mole (mol)</td>
+    </tr>
+    <tr>
+      <td>Luminous intensity</td>
+      <td>candela (cd)</td>
+    </tr>
+    <tr>
+      <td>Force</td>
+      <td>newton (N), dyne (dyn), poundforce (lbf), kip</td>
+    </tr>
+    <tr>
+      <td>Energy</td>
+      <td>joule (J), erg, Wh, BTU, electronvolt (eV)</td>
+    </tr>
+    <tr>
+      <td>Power</td>
+      <td>watt (W), hp</td>
```

Comment:
horsepower (hp)?

Reply:
Right, it refers to horsepower

Reply:
I meant for you to write it in :smile: to keep it consistent with watt.

Reply:
Done.